pkg/middleware/middleware.go

```diff
@@ -59,8 +60,22 @@ func Process(ctx *quaycontext.QuayRegistryContext, quay *v1.QuayRegistry, obj cl
 			return dep, nil
 		}
 
+		// Set env variable overrides
```

Comment:
You are not setting env variables for the "quay-database" deployment (there is a return on line 60 above). Is that on purpose?